webform_civicrm.info.yml

```diff
@@ -1,8 +1,8 @@
-name: 'CiviCRM Webform'
+name: 'Webform CiviCRM'
 type: module
-description: 'CiviCRM and Webform integration'
-core_version_requirement: ^8.7.7 || ^9
+description: 'Webform CiviCRM Integration'
+core_version_requirement: ^8.8 || ^9
 package: 'Custom'
 dependencies:
-  - webform
+  - drupal:webform (>= 6.0)
```

Comment:
Do we want `(>= 6.0)`? Isn't 6.0 only D9 and not D8. Or is it 8.8? We'd also need to update the `composer.json` as well to require ^6.0

I think it's fine just having `drupal:webform` and the composer constraint allowing `^5 || ^6`

Reply:
Webform module 5.x-8.x and 6.x are both:
`core_version_requirement: ^8.8 || ^9`

Reply:
I've added a commit to update `composer.json`